drivers/video/ov5640.c

```diff
@@ -650,6 +875,11 @@ static int ov5640_stream_start(const struct device *dev)
 static int ov5640_stream_stop(const struct device *dev)
 {
```

Comment:
For DVP mode, I think we need to implement stream _start / stop_ also. Pipeline should be capable to stop and restart. Could you refer to the Linux implementation [here](https://elixir.bootlin.com/linux/v6.11.1/source/drivers/media/i2c/ov5640.c#L1821)